packages/venia-concept/_buildpack/create.js

```diff
@@ -28,7 +28,10 @@ function createProjectFromVenia({ fs, tasks, options }) {
         'validate-queries',
         'watch'
     ];
-    const scriptsToInsert = {};
+    const scriptsToInsert = {
+        storybook: 'start-storybook -p 9001 -c src/.storybook',
+        'storybook:build': 'build-storybook -c src/.storybook -o storybook-dist'
```

Comment:
@zetlen - I opted to move these scripts here from `package.json` in venia-concept, since they weren't executable if attempting to run inside the mono (this assumes `venia-ui` is in `node_modules`). This set off some snapshot failures and I didn't feel confident in which method is best. Couple clarifying questions:

1. Was it okay to have failing scripts in `venia-concept`?
2. Am I using `scriptsToInsert` as intended?